tests/Schema/ComparatorTest.php

```diff
@@ -1064,39 +1058,6 @@ public function assertSchemaSequenceChangeCount(
         self::assertCount($removeSequenceCount, $diff->removedSequences);
     }
 
-    public function testDiffColumnPlatformOptions(): void
```

Comment:
Changes in platform options are no longer exposed via the column diff as they were never used anywhere except for this test.